src/python/foglamp/update_statistics_history.py

```diff
@@ -0,0 +1,139 @@
+"""
+The following piece of code takes the information found in the statistics table, and stores it's delta value 
+(statistics.value - statistics.prev_val) inside the statistics_history table. To complete this, SQLAlchemy will be 
+used to execute SELECT statements against statistics, and INSERT against the statistics_history table.  
+"""
+import sqlalchemy
+import time
+
+__author__ = "Ori Shadmon"
+__copyright__ = "Copyright (c) 2017 OSI Soft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+# Set variables for connecting to database
+_db_type = "postgres"
+_user = "foglamp"
+_db_user = "foglamp"
+_host = "192.168.0.182"
+_db = "foglamp"
```

Comment:
All of the above variables should be deleted. The connection string is dbname='foglamp'

Please change purge.py too and include it in this branch.